test/unittests/stack_test.cpp

```diff
@@ -130,3 +131,94 @@ TEST(stack, clear_on_empty)
     Stack<char> stack;
     stack.clear();
 }
+
+TEST(operand_stack, construct)
+{
+    OperandStack stack{0};
+    EXPECT_THAT(stack.size(), 0);
```

Comment:
Why use `EXPECT_THAT` in these simple comparisons?

Reply:
I though you want to migrate to `EXPECT_THAT()` so I started using it.

Reply:
I don't mind personally. Not sure about impact/benefit/difference. For the more complex return types it looks like a good solution as you've suggested.